src/ftest.jl

```diff
@@ -1,3 +1,10 @@
+struct SingleFTestResult{T<:Real}
+    df1::T
+    df2::T
```

Comment:
Always integers for classical ANOVA, but e.g. the Greenhouse-Geiser and Huyn-Feldt corrections work be changing the df to potentially non integers. Not sure if we want to leave room to support that in future releases.

Reply:
Hmm, I guess changing the type later wouldn't be considered as breaking, right? We already use `Int` for `FTest` too FWIW.

Reply:
But yes, I agree it would be great to print numbers that are integers as integers, whether or not the underlying type is `Int`.

Reply:
I think this has gone from Int to parametric in the past. Maybe because of StatsModels but it seems we gave up on integrating with that here and also FTestResult{N} uses Int => I made it Int.